### PR TITLE
feat(academy): add quiz part 1.1

### PR DESCRIPTION
**Pull request by MathieuConte**
Add 6 questions to the part 1.1 

- How are Bounded digital resources classified in terms of size?
- What is the main difference between Infrastructure Services and Digital Resource Processing Services ?
- What do Verifiable Credentials (VCs) bring to the OKP4 protocol?
- What is the fundamental difference between Verifiable Claims (VCs) and Verifiable Presentations (VPs)?
- How does Ontology contribute to solving challenges related to the diversity of digital resources in the OKP4 protocol?
- How do Verifiable Claims and Ontology interact in the OKP4 protocol?

Good luck :D 

FYI @JulienMassonnat @amimart @emmanuelaldeguer @tpelliet 

## Summary by CodeRabbit

- **New Features**
	- Introduced an interactive quiz feature to the Academy resources section for enhanced learning engagement.

### CODE REVIEW

File: docs/academy/resources-quiz.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      },
+      {
+        "question": "What do Verifiable Credentials (VCs) bring to the OKP4 ?",
+        "questionType": "text",
+        "answerSelectionType": "single",
+        "answers": [
+          "A method for storing resources directly on the blockchain.",
+          "A way to ensure the sovereignty of digital resources.",
+          "A mechanism for identifying resources without centralized control.",
+          "A means to describe specific properties of a resource"
```
Comment (ccamel) [inline docs/academy/resources-quiz.json:43]: I'd suggest adding a little more specificity to this mechanism:

```suggestion
          "A means to describe specific properties of a resource securely and in a verifiable manner"

```

what do you think?

File: docs/academy/resources-quiz.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          "By categorizing resources rigidly to facilitate understanding.",
+          "Ontology has no impact on the diversity of digital resources."
+        ],
+        "correctAnswer": "1",
+        "messageForCorrectAnswer": "Correct answer. Good job.",
+        "messageForIncorrectAnswer": "Incorrect answer. Please try again.",
+        "point": "1"
+      },
+      {
+        "question": "How do Verifiable Claims and Ontology interact in the OKP4 protocol?",
```
Comment (ccamel) [inline docs/academy/resources-quiz.json:81]: It's a very subtle question. If you asked me to choose, I'd go for answer 1: "Verifiable Claims are an integral part of Ontology in the OKP4 protocol."

Verifiable Claims (or Verifiable Credentials) form the foundational elements of the ontology by providing the substantiated assertions about every resources of the dataverse. The relationships and connections between resources in the ontology are always established and enriched through these claims, serving as the backbone of knowledge support for the ontology, in my vision.

**Review by ccamel — CHANGES_REQUESTED** [review]
Great initiative! :smiley: 
I've made a few comments that should be taken into account. Some of the quiz questions are tricky and need special attention.

File: docs/academy/resources-quiz.json (hunk 2 above)
Comment (MathieuConte) [reply docs/academy/resources-quiz.json:82]: Oh ok! I didn't see it that way! I saw two distinct elements that act in a complementary manner, but Marine also explained it to me!

I will change option 1 to put something even more incorrect!

Thanks !

Comment (ccamel) [reply docs/academy/resources-quiz.json:82]: OK, then. Let's go this way.

**Review by ccamel — APPROVED** [review]
Good! :+1:

**Review by MdechampG — APPROVED** [review]
👍🏻

**Pull request merged**